flytekit/core/type_engine.py

```diff
@@ -1309,6 +1311,11 @@ def convert_json_schema_to_python_class(schema: dict, schema_name) -> Type[datac
 def _get_element_type(element_property: typing.Dict[str, str]) -> Type[T]:
     element_type = element_property["type"]
     element_format = element_property["format"] if "format" in element_property else None
+
+    if type(element_type) == list:
```

Comment:
How does this work for `Union`?

Reply:
[marshmallow](https://github.com/marshmallow-code/marshmallow/issues/1191#issuecomment-480831796) doesn't support union type